### DISCUSSION TITLE
Argo CD suddenly stopped detecting new applications. Whilst still syncing existing one nornmally

### DISCUSSION THREAD

**Discussion by bob-bn** [argoproj/argo-cd#15790] [General]
Hello, 

our Argo CD ( 2.6.0 ) **suddenly** stopped syncing new applications, it was totally fine and we have not changed any configuration. Syncing works well however. 

I am unable to know / find any logs that hints to the problem. 

When I apply the application yaml manually, everything kicks in normally there after, and it get auto synced and all...

What could be or where could be the problem here ?

Best,

**Comment by bowale01** [comment]
This might be a bug since the manual sync is work. can you flag the disable autosyn and enable it again, this should resolve the issue i guess if it is not a bug
